src/index.ts

```diff
@@ -0,0 +1,179 @@
+import get from "lodash/get";
+import { parse } from "./parsers";
+import {
+  ConfigOptions,
+  InjectedProps,
+  Config,
+  ResolvedSchema,
+  isStringEnumConfig,
+  isNumberConfig,
+  isCustomConfig,
+  isBooleanConfig,
+  isStringConfig,
+  StringConfig,
+  StringEnumConfig,
+  NumberConfig,
+  BooleanConfig,
+  CustomConfig,
+  AdminField,
+  AdminFields,
+  GenericAdminFields,
+  NamespacedUseConfigReturnType,
+} from "./types";
+import { createUseAdminConfig } from "./createUseAdminConfig";
+import { createUseConfig } from "./createUseConfig";
+
+export {
+  // Options
+  ConfigOptions,
+  // Configs
+  Config,
+  StringConfig,
+  StringEnumConfig,
+  NumberConfig,
+  BooleanConfig,
+  CustomConfig,
+  // Typeguards
+  isStringEnumConfig,
+  isNumberConfig,
+  isCustomConfig,
+  isBooleanConfig,
+  isStringConfig,
+  // useConfigAdmin.fields
+  AdminField,
+  AdminFields,
+  GenericAdminFields,
+};
```

Comment:
Can commented `Options`, `Configs`, `Typeguards` etc be removed from here?

Reply:
This was to organized a bit the exports (and avoid to forgot some ^^)

Reply:
(It's a title, not a commented type)